lld/test/ELF/invalid/x86-64-tls-reloc.s

```diff
@@ -0,0 +1,36 @@
+# REQUIRES: x86
+# RUN: llvm-mc -filetype=obj -triple=x86_64 %s -o %t.o
+# RUN: not ld.lld %t.o -o /dev/null 2>&1 | FileCheck %s
```

Comment:
`-DFILE=%t.o` (see systemz-reloc-pcdbl.s as an example) and expand `{{.*}}` to `[[FILE]]` below

Reply:
Updated. Thanks.